core/src/main/java/org/apache/brooklyn/entity/group/SshCommandMembershipTrackingPolicy.java

```diff
@@ -62,6 +69,38 @@
     public static final String EVENT_TYPE = "EVENT_TYPE";
     public static final String MEMBER_ID = "MEMBER_ID";
 
+    public static enum ExecutionTarget {
+        ENTITY,
+        MEMBER,
+        ALL_MEMBERS;
+
+        private static Converter<String, String> converter = CaseFormat.LOWER_CAMEL.converterTo(CaseFormat.UPPER_UNDERSCORE);
+
+        public static ExecutionTarget fromString(String name) {
+            Maybe<ExecutionTarget> parsed = tryFromString(name);
+            return parsed.get();
+        }
+
+        public static Maybe<ExecutionTarget> tryFromString(String name) {
```

Comment:
But the methods here are _for_ `TypeCoercions` to work... If I use it, won't there be an infinite recursion?